### ISSUE TITLE
Add "Usage with Svelte" to documentation

### ISSUE THREAD

**Issue by mhatvan** [duetds/date-picker#62]
**Is your feature request related to a problem? Please describe.**
I am working on a client project with Svelte and this date picker would be a great replacement for the current one. 

**Describe the solution you'd like**
Add instructions of how to get Duet Date Picker up and running with Svelte.

**Comment by WickyNilliams** [comment]
Hey. Would you be able to make a PR for this? I have zero experience with Svelte, so I would not be able (or confident!) writing instructions

**Comment by mhatvan** [comment]
Yes, I can try to get it up and running with Svelte and provide a PR once I find the time!

**Comment by WickyNilliams** [comment]
Thanks, appreciate it!

**Comment by mhatvan** [comment]
I tried to follow the instructions similar to Vue.js but was unable to get neither a starter template for Svelte nor Sapper to work with duetds date-picker. It would probably require a person more familiar with web components to tackle this.

**Comment by Wolfr** [comment]
Since Svelte is a compiler and has no virtual DOM; and any Svelte project itself can compile as a web component, it would be a bit weird to try and integrate Duet as a web component.

A correct way to integrate would be to use Svelte to import the Typescript and then build a Svelte component with the right props from there.

**Comment by WickyNilliams** [comment]
Not sure how svelte not having a virtual DOM changes anything? A web component doesn't require virtual DOM. Stencil uses one, but its an implementation detail from an external perspective.

Maybe some code can help illustrate what you mean. As I said above I have zero experience with svelte so maybe I'm missing something
